executor/executor.go

```diff
@@ -943,7 +943,7 @@ func doLockKeys(ctx context.Context, se sessionctx.Context, lockCtx *kv.LockCtx,
 	if err != nil {
 		return err
 	}
-	return txn.LockKeys(ctx, lockCtx, keys...)
+	return txn.LockKeys(context.WithValue(ctx, sessionctx.ConnID, se.GetSessionVars().ConnectionID), lockCtx, keys...)
```

Comment:
Could we use https://github.com/pingcap/tidb/blob/fe764a97a3966e19129291e099de3659e0e38511/sessionctx/context.go#L138 directly?

Reply:
Maybe it is better to change its name.